trade-finance-manager-ui/templates/case/tasks/task.njk

```diff
@@ -4,11 +4,17 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle -%}
+  Deal {{ deal.details.ukefDealId }} - Task {{ task.title }}
```

Comment:
GEF deals?

Reply:
Yes, it works for GEF, other deals, and all Facilities.